hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/chaos/TestProbability.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.chaos;
+
+import org.apache.commons.lang3.RandomUtils;
+
+/**
+ * Class to keep track of test probability.
+ */
+public class TestProbability {
+  private int pct;
+
+  private TestProbability(int pct) {
+    this.pct = pct;
+  }
+
+  public boolean isTrue() {
+    return (RandomUtils.nextInt() * pct / 100) == 1;
```

Comment:
Can you please explain how this should work?

I would assume that `pct = 100` means 100% probability, ie. it should always return `true`.  This requires `nextInt=1`, but `RandomUtils.nextInt()` returns values between 0 and `Integer.MAX_VALUE`, so it happens only with very low probability.  On the other hand, for `pct = 1` this returns `true` for any `nextInt` in `[100, 199]`, which has higher probability than for `pct  = 100`, but still lower than 1%.

Reply:
Sorry, my bad. This is a coding error. I had quickly tried something in the notepad. let me updated the patch soon.

Reply:
Thanks for the review @adoroszlai, this is fixed as part of the update to this pull request.

Reply:
Thanks.

Reply:
@adoroszlai are you +1 on the latest changes?

Reply:
@arp7 @mukul1987 Yes, the new `isTrue` logic is fine.  (I would add a special case for 0 to avoid division by zero, and check for negative values in the constructor, but these are not "normal" use cases I guess.)

But I didn't want to +1 the whole change, since I'm not familiar with most of it...  (and my +1 is not-binding anyway).

Reply:
Non-binding +1s are still very useful. I will review.